src/components/proofs/DesktopProofCategories.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  CategoriesComponentProps,
+  CategoriesTitles,
+  categories,
+} from 'models/Categories'
+import classNamesToString from 'helpers/classNamesToString'
+import classnames, {
+  alignItems,
+  backgroundColor,
+  borderRadius,
+  display,
+  flexDirection,
+  gap,
+  height,
+  inset,
+  justifyContent,
+  opacity,
+  padding,
+  position,
+  stroke,
+  transitionProperty,
+  width,
+} from 'classnames/tailwind'
+
+const desktopMenuWrapper = classnames(
+  display('md:flex', 'hidden'),
+  flexDirection('flex-col'),
+  position('sticky'),
```

Comment:
A tricky way to position this element, simple relative or absolute will never work, because we place the menu inside the `Scrollbar`. The trick with scrollbar is that it's relative to wrapper `Card`, wrapping `Scrollbar` in another `div` will break it